biz/handler/admin.go

```
@@ -25,7 +25,7 @@ func HandleBanInfoHash(_ context.Context, c *app.RequestContext) {
 	}
 	manager := peer.GetPeerManager()
 	for _, infoHash := range req.Hash {
-		manager.BanInfoHash(infoHash)
+		manager.BanInfoHash(ctx, infoHash)
```

golangci-lint (1.62.2)

Error return value of `manager.BanInfoHash` is not checked

(errcheck)